### PR TITLE
Add monit docs to Upgrade Guide

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Add documentation as a reminder to shutdown/restart monit during the upgrade process.

# Impacts
Upgrade Guide

# Delete branch after merge
YES 

# NEWS file entries
## New Features
* Added monit instruction to upgrade guide

### CODE REVIEW

File: UPGRADE.asciidoc [hunk 1]
```diff
@@ -34,6 +34,22 @@ You can take a backup of the pf directory with the following command:
...
+Shutdown monit
+^^^^^^^^^^^^^^
+
+Before starting the upgrade shut down monitoring to prevent alerts from generating false-positives. To see if monit is installed and running issue a monit summary.
+
+  monit summary
+
+If monit is installed and running, shut it down with
+
+  systemctl stop monit
```
Comment (fdurand) [inline UPGRADE.asciidoc:46]: then disable monit
systemctl disable monit

File: UPGRADE.asciidoc [hunk 2]
```diff
@@ -34,6 +34,22 @@ You can take a backup of the pf directory with the following command:
...
+
+When the upgrade is complete restart monit back up with
+
```
Comment (fdurand) [inline UPGRADE.asciidoc:49]: systemctl enable monit

**Comment by nqb** [comment]
@fdurand : I think we should fix #3499 in place of adding disable/enable in docs.

**Comment by nqb** [comment]
I will close this issue and integrate @satkunas changes into #5926

**Pull request closed without merging**